test/e2e/node/security_context.go

```diff
@@ -252,10 +252,11 @@ func testPodSELinuxLabeling(f *framework.Framework, hostIPC bool, hostPID bool)
 	pod.Spec.SecurityContext.SELinuxOptions = &v1.SELinuxOptions{
 		Level: "s0:c0,c1",
 	}
-
 	f.TestContainerOutput("Pod with same MCS label reading test file", pod, 0, []string{testContent})
+
```

Comment:
ditto

Reply:
This one is actually related. As I wrote in commit message, I am also trying to improve readability for this particular piece of code, thus the empty line (to distinguish this is a separate test case).